src/libraries/System.Net.Http.WinHttpHandler/src/System/Net/Http/NetEventSource.WinHttpHandler.cs

```diff
@@ -6,7 +6,5 @@
 namespace System.Net
 {
     [EventSource(Name = "Private.InternalDiagnostics.System.Net.Http.WinHttpHandler", LocalizationResources = "FxResources.System.Net.Http.WinHttpHandler.SR")]
-    internal sealed partial class NetEventSource : EventSource
-    {
-    }
+    internal sealed partial class NetEventSource { }
```

Comment:
I am just curious why ": EventSource" is not needed here anymore?

Reply:
It was never needed since it's defined in the common version of this partial class.
This is more style than anything else - other derived definitions don't duplicate it.

Reply:
I see it still duplicating in `System.Net.Http/src/System/Net/Http/NetEventSource.Http.cs`, and coincidentally it was the one I compared to when I was looking how is it in other event sources 😄

Reply:
Hmm... maybe I'm lying... it looks like it's [50/50](https://grep.app/search?q=class%20NetEventSource).

I'll change them all to use the same pattern.